recipes/mergedeep/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mergedeep" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "dd95396caea02ec43e86309c30fb15168ac0be8f3df9f9bc7d3d3e49eef48ed8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clarketm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
Can you use the canonical form of this from the example:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35